bazel/linux/templates/runner.template.sh

```diff
@@ -157,6 +161,7 @@ echo 1>&2 "Running emulator..."
 {code}
 
 estatus="$?"
+test "$estatus" == 0 && estatus=$(cat "$EXIT_STATUS_FILE")
```

Comment:
`$(< "$EXIT_STATUS_FILE")` no need for cat in bash.

Reply:
also, messages below should be updated: they should clearly specify if the emulator failed, one of the tests inside the emulator (which? ideally!).

Reply:
Fixed in next revision.